binaries/seesaw_cli/main.go

```diff
@@ -184,9 +188,11 @@ func interactive() {
 		if cmdline == "" {
 			continue
 		}
+		terminal.Restore(syscall.Stdin, oldTermState)
```

Comment:
what's the symptom if you don't do this? Especially it's only changing `syscall.Stdin`. `fmt.Print` prints to `stdout` by default.

Reply:
The symptom is fmt.Println doesn't go back to the begining of a line. terminal.Restore applies to whole terminal not just stdin.

Reply:
interesting to know that..